laos_climate_change/data_loaders.py

```diff
@@ -0,0 +1,27 @@
+# Verify world shapefile
+from os.path import exists
+from urllib.request import urlretrieve
+
+path_to_file = "C:/Users/camil/Dropbox/Servicios/OECD-GGGI/laos-climate-change/laos_climate_change/wb_countries_admin0_10m.zip"
```

Comment:
Use a relative import here, since my computer doesn't have `C:/users/camil` : )

You can just write; `../data/shapefilies/data/wb_countries_admin0_10m.zip`. The `..` means "one level up".